modules/oe_theme_content_event/src/Plugin/ExtraField/Display/DescriptionExtraField.php

```diff
@@ -88,13 +88,16 @@ public function getLabel() {
    * {@inheritdoc}
    */
   public function viewElements(ContentEntityInterface $entity) {
+    // If we have no renderable text and image wee don't need a title.
+    $text = $this->getRenderableText($entity);
+    $title = !empty($text[0]['#text']) || !$entity->get('oe_event_featured_media')->isEmpty() ? $this->getRenderableTitle($entity) : '';
```

Comment:
As I can see in \Drupal\oe_theme_content_event\Plugin\ExtraField\Display\DescriptionExtraField::addFeaturedMediaThumbnail implementation we have to check not only if the field is empty but also if the media still exists and the user still has access to media. Maybe it makes sense to implement a separate method for checking if the image can be exposed.